draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -541,70 +539,69 @@ Flow Identifier:
 : Flow identifier to demultiplex different data flows on the same QUIC
 connection.
 
-RTP/RTCP Payload:
+RTP Payload:
 
-: Contains the RTP/RTCP payload; see {{fig-rtp-stream-payload}}
+: Contains the RTP payload; see {{fig-rtp-stream-payload}}
 
 The payload in a QUIC STREAM frame starts with the flow identifier followed by one or
-more RTP/RTCP payloads. All RTP/RTCP payloads sent on a STREAM frame MUST belong to
+more RTP payloads. All RTP payloads sent on a STREAM frame MUST belong to
 the RTP session with the same flow identifier.
 
-Each payload begins with a length field indicating the length of the RTP/RTCP
+Each payload begins with a length field indicating the length of the RTP
 packet, followed by the packet itself, see {{fig-rtp-stream-payload}}.
 
 ~~~
-RTP/RTCP Payload {
+RTP Payload {
   Length(i),
-  RTP/RTCP Packet(..),
+  RTP Packet(..),
 }
 ~~~
-{: #fig-rtp-stream-payload title="RTP/RTCP payload for QUIC STREAM frames"}
+{: #fig-rtp-stream-payload title="RTP payload for QUIC STREAM frames"}
 
 Length:
 
 : A QUIC variable length integer (see {{Section 16 of !RFC9000}}) describing the
-length of the following RTP/RTCP packets in bytes.
+length of the following RTP packets in bytes.
 
-RTP/RTCP Packet:
+RTP Packet:
 
-: The RTP/RTCP packet to transmit.
+: The RTP packet to transmit.
 
 ### Media Frame Cancellation
 
 QUIC uses RESET\_STREAM and STOP\_SENDING frames to terminate the sending part
 of a stream and to request termination of an incoming stream by the sending
 peer respectively.
 
-A RoQ sender can use RESET\_STREAM if it knows that a packet, which was not yet
-successfully and completely transmitted, is no longer needed.
-
 A RoQ receiver that is no longer interested in reading a certain partition of
 the media stream can signal this to the sending peer using a STOP\_SENDING
 frame.
 
+If a RoQ sender discovers that a packet is no longer needed and knows that the packet that has not yet been successfully and completely transmitted, it can use RESET\_STREAM to tell the RoQ receiver that the RoQ sender is discarding the packet.
```

Comment:
```suggestion
If a RoQ sender discovers that a packet is no longer needed and knows that the packet has not yet been successfully and completely transmitted, it can use RESET\_STREAM to tell the RoQ receiver that the RoQ sender is discarding the packet.
```